pkg/config/otlp.go

```diff
@@ -26,6 +27,10 @@ func SetupOTLP(config Config) {
 	config.BindEnv(ExperimentalOTLPHTTPPort, "DD_OTLP_HTTP_PORT")
 	config.BindEnv(ExperimentalOTLPgRPCPort, "DD_OTLP_GRPC_PORT")
 
+	// NOTE: This only partially works.
+	// The environment variable is also manually checked in pkg/otlp/config.go
+	config.BindEnv(ExperimentalOTLPTagCardinalityKey, "DD_OTLP_TAG_CARDINALITY")
```

Comment:
`dogstatsd_tag_cardinality` defaults to `low`, should we do the same here?

Reply:
This happens here
https://github.com/DataDog/datadog-agent/blob/a539e9652817a9cac232a5c80f7a98b2bac5de80/pkg/otlp/internal/serializerexporter/exporter.go#L40

Reply:
Here we are reading an empty string, though:
https://github.com/DataDog/datadog-agent/blob/922d2424a2ea6c722c84d5798e0f29c366aae861/pkg/otlp/config.go#L152
From what you say I assume the later in the code the default "wins" over the empty string, but I wasn't 100% sure :)

Reply:
ah, I think you are right :( and I think "" is interpreted as "whatever `dogstatsd_tag_cardinality` has, based on https://github.com/DataDog/datadog-agent/blob/a539e9652817a9cac232a5c80f7a98b2bac5de80/pkg/tagger/global.go#L282-L284 We should fix this

Reply:
#10634